debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -350,6 +342,34 @@ public class OracleConnectorConfig extends HistorizedRelationalDatabaseConnector
             .withDescription("Sets the specific archive log destination as the source for reading archive logs." +
                     "When not set, the connector will automatically select the first LOCAL and VALID destination.");
 
+    public static final Field ARCHIVE_DESTINATION_NAME = Field.create("archive.destination.name")
+            .withDisplayName("Name of the archive log destination to be used for reading archive logs")
+            .withType(Type.STRING)
+            .withWidth(Width.MEDIUM)
+            .withImportance(Importance.LOW)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTION_ADVANCED, 32))
+            .withDescription("Sets the specific archive log destination as the source for reading archive logs." +
+                    "When not set, the connector will automatically select the first LOCAL and VALID destination.");
```

Comment:
We probably should make sure this gets highlighted in the release notes. For XStream and OpenLogReplicator users, they will not have defined this value at all and in most production environments, this will be required to avoid potential a miscalculation.

Reply:
Well, I was tempted to put it as `required` but I was in doubt. Do you think is it an option?

Reply:
I would not set it as required because the connector is smart enough to resolve the value if only 1 VALID and LOCAL log destination is configured.  

For OpenLogReplicator and XStream users, they'll need to check their `V$ARCHIVED_DEST_STATUS` table and confirm if there are multiple rows that match `STATUS=VALID and TYPE=LOCAL`.  If only a single row, they can safely leave this option as the default. If they have multiple, they'll need to confer with the DBA to know which configured destination should be used.

Reply:
Or should we do this check https://github.com/debezium/debezium/blob/b6e45beab1d04683d1ae11bc1879770f936b9f0d/debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerStreamingChangeEventSource.java#L932 also for XStream and OpenLogReplicator?

Reply:
> We could; however, add this to the Snapshotter code as a sanity check before fetching the oldest available SCN from the logs, wdyt?

This is what I meant.